problem_builder/public/js/translations/de_DE/textjs.js

```diff
@@ -30,7 +30,11 @@
       "Ergebnisse am <%= creation_time %> (<%= seconds %> second) abgerufen", 
       "Ergebnisse am <%= creation_time %> (<%= seconds %> second) abgerufen."
     ], 
-    "The report is currently being generated\u2026": "Der Report wird gerade zusammengestellt\u2026"
+    "The report is currently being generated\u2026": "Der Report wird gerade zusammengestellt\u2026", 
+    "You have used {num_used} of 1 submission.": [
+      "Sie haben {num_used} von einer Einreichung verwendet.", 
+      "Sie haben {num_used} von {max_attempts} Einreichungen verwendet.        "
```

Comment:
Why is there whitespace after the fullstop within the string?

Reply:
@aayushagra Regarding white space issues that you have identified, All highlighted areas belong to auto generated files. You can simply ignore this since we are not adding/deleting anything in these files manually.